node-export-server/systemd/node-export.service

```diff
@@ -4,8 +4,8 @@ After=syslog.target network.target
 
 [Service]
 Type=single
-User=vagrant
-WorkingDirectory=/home/vagrant/hoot/node-export-server
+User=SERVICE_USER
+WorkingDirectory=HOOT_HOME/node-export-server
```

Comment:
`SERVICE_USER` and `HOOT_HOME` are replaced in the provisioning script above with `sed`